src/bosh_azure_cpi/lib/cloud/azure/models/config.rb

```diff
@@ -106,31 +107,43 @@ class RegistryConfig
     attr_reader :endpoint, :user, :password
 
     def initialize(registry_config_hash)
-      @config = registry_config_hash
-
-      @endpoint = @config['endpoint']
-      @user = @config['user']
-      @password = @config['password']
+      @endpoint = registry_config_hash['endpoint']
+      @user = registry_config_hash['user']
+      @password = registry_config_hash['password']
     end
   end
 
   class AgentConfig
     def initialize(agent_config_hash)
-      @config = agent_config_hash
+      @config_hash = agent_config_hash
     end
 
     def to_h
-      @config
+      @config_hash
     end
   end
 
   class Config
+    include Singleton
+    include Helpers
```

Comment:
what is the purpose of  `include Helpers`?

Reply:
removed the extra include.